LHM/Problems/beakjoon/1110.py

```diff
@@ -0,0 +1,25 @@
+#https://www.acmicpc.net/problem/1110
+import math
+inputVal = int(input())     #26
+in1 = math.floor(inputVal/10)   #2
```

Comment:
```suggestion
in1 = inputVal//10
```
The operator // can return the quotient only (useful during integer calculation)